app/models/channel.rb

```diff
@@ -57,10 +58,15 @@ class Channel < ApplicationRecord
     in: %w[domain_not_found connection_failed too_many_redirects timeout no_txt_records token_incorrect_dns token_not_found_dns token_not_found_public_file no_https]
   }, allow_nil: true
 
+  validates :public_name, uniqueness: true, allow_nil: true
+  # validates :public_identifier, uniqueness: true, presence: true
```

Comment:
Comment

Reply:
In ca you were wanting to uncomment this for uniqueness, nevermind. But you can also add an index in the migration to enforce uniqueness.

Reply:
This validation will work fine once I run the rake task to backfill values.  Until I do that, though, the validation will cause a bunch of errors.  I'll uncomment it in a future PR once the task has been run.